src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -122,6 +155,40 @@ public async Task<Response<ConfigurationSetting>> SetAsync(ConfigurationSetting
             }
         }
 
+        public async Task<Response<ConfigurationSetting>> SetAsync(string key, string value, CancellationToken cancellation = default)
```

Comment:
Could this method create a `ConfigurationSetting` and call into the other overload? That might reduce the duplicated code in this method.

Reply:
:| Thank you!